standalone-examples/src/main/java/io/pravega/example/noop/SimpleReader.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.example.noop;
```

Comment:
Consider moving `SimpleReader` to another package.

Reply:
I'd like to keep it here. All other samples are self-contained in a single package. If this does get used elsewhere we should move it at that time.